assets/app/views/_overview-deployment.html

```diff
@@ -106,7 +107,7 @@
           </a>
         </div>
 
-        <div column class="scaling-controls">
+        <div column class="scaling-controls" ng-if="!hpa.length">
```

Comment:
so does this mean if the rcs and dcs load before the hpas that you might see the scaling controls appear and then disappear?